ui/src/CurrentUser.ts

```diff
@@ -99,7 +99,9 @@ export class CurrentUser {
     };
```

Comment:
FYI: your commits aren't linked to your GitHub account. See the email in https://github.com/gotify/server/pull/723/commits/eaa19ddcb110296b7fd49e8589f2c5a3b22b2b55.patch